bokeh/palettes.py

```diff
@@ -1185,6 +1202,10 @@
     "Set3"     : Set3,
 }
 
+bokeh = {
+    "Bokeh": BokehPalette
+}
```

Comment:
I wonder if we really need this since there is only one palette... We need to add this to `all_palettes` below,  in any case, but maybe we *only* do that, and don't create this named dict here?

Reply:
so if I remove this, it breaks the `.. bokeh-palette-group:: bokeh` on line 176. I bet this is also why 'colorblind' has its own group (ln 1224) despite also being a single-member group. I am open to further suggestions, though.

Reply:
Ah, well that's slightly annoying but I think for now just adhering to the current convention is simplest.